### PR TITLE
Allow for analysis under rebar3_hank and rebar3_lint

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
This'll require `elvis_core`'s [most recent](https://github.com/inaka/elvis_core/pull/202) pull [requests merged](https://github.com/inaka/elvis_core/pull/203), tagged and published, since local development benefited from `_checkouts` 😄.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,15 +14,12 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        otp_vsn: [20, 21, 22, 23, 24]
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/ci.yml:17]: `hank` imposes 21+

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -14,15 +14,12 @@ jobs:
...
+        otp_vsn: [21, 22, 23, 24]
         os: [ubuntu-latest]
     steps:
       - uses: actions/checkout@v2
       - uses: erlef/setup-beam@v1
         with:
           otp-version: ${{matrix.otp_vsn}}
           rebar3-version: '3.14'
-      - run: rebar3 xref
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/ci.yml:25]: Yay, simplification.

File: src/mixer.erl [hunk 3]
```diff
@@ -22,6 +22,15 @@
 
 -export([parse_transform/2]).
 -ignore_xref(parse_transform/2).
+% ... since there's no behaviour for parse transformations
```
Comment (paulo-ferraz-oliveira) [inline src/mixer.erl:25]: 😭

File: test/failure_test.erl [hunk 4]
```diff
@@ -3,18 +3,19 @@
 -include_lib("kernel/include/file.hrl").
 -include_lib("eunit/include/eunit.hrl").
 
--define(EXPORTS(Mod), Mod:module_info(exports)).
```
Comment (paulo-ferraz-oliveira) [inline test/failure_test.erl:6]: Yay, `hank`.

File: test/import_test.erl [hunk 5]
```diff
@@ -2,6 +2,9 @@
 
 -include_lib("eunit/include/eunit.hrl").
 
+% call to :module_info
+-elvis([{elvis_style, invalid_dynamic_call, #{ ignore => [{import_test, exports, 1}] }}]).
```
Comment (paulo-ferraz-oliveira) [inline test/import_test.erl:6]: Actually, this also requires a recent fix to `elvis_core`, but that one's already merged to `master`.

**Comment by paulo-ferraz-oliveira** [comment]
Bump, @elbrujohalcon.

**Review by elbrujohalcon — APPROVED** [review]

**Pull request merged**